Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -211,6 +229,40 @@ function delete(obj)
                 rethrow(err)
             end
         end
+        
+        function val = set_wheel_ip(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', val, obj.wheel_pin);
+            if isempty(obj.wheel)
+                if ~isempty(err)
+                    rethrow(err)
+                end
+                val = 'No Server';
+                return
+            end
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
+        function val = set_wheel_pin(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', obj.wheel_ip, val);
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
```

Comment:
Should this also set val to nan if isempty(obj.wheel)?

Reply:
Then you run into a catch 22 situation where setting the host breaks because the pin is NaN and setting the pin breaks because the host is No Server.

Reply:
can't connect_driver only connect when pin and host are valid?